tests/optimize/test_objective.py

```diff
@@ -218,6 +218,17 @@ def test_directory(self):
         self.assertAlmostEqual(x["('1', '1')"]['sigma'], self.sigma.value)
         self.assertAlmostEqual(x["('1', '1')"]['rmax'], 2.7)
 
+        sim = self.simulation.run(self.target, self.potentials, self.directory.name)
```

Comment:
We should also add a test (probably in its own test function, or else maybe in `test_compute`) that checks that the `gradient()` method works properly.